.github/PULL_REQUEST_TEMPLATE/guide_pull_request_template.md.md

```diff
@@ -0,0 +1,11 @@
+# Guide Outline
+
+> Provide a one or two paragraph outline of your guide. You should mention the overall topic, and what you expect learners to acheive when completing your guide.
+
+## Link to Proposal
+
+> Include a link to the related proposal on Kafé (e.g. <https://dev.builderdao.io/vote/my-proposal-slug>)
```

Comment:
link to the proposal is auto-generated by the GitHub action.  for example: 
https://github.com/TheBuilderDAO/kafe/pull/158#issuecomment-1099556450
would appreciate some styling feedback on this one actually @pyr0gan @dgamboa

Reply:
I'm not sure what you mean by styling feedback here, @Necmttn 🤔

Reply:
<img width="887" alt="image" src="https://user-images.githubusercontent.com/5212808/164427657-d0e443ec-50af-46fd-b22f-75c33dab2666.png">
as the wording and utilization of emojis etc.